### PR TITLE
(WIP) feat: added examples to vcs/*

### PR DESCRIPTION
**Pull request by SuperVrata**
## Description
Added examples to the VCS capabilities

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: capabilities/vcs/user-repos/profile.supr [hunk 1]
```diff
@@ -45,4 +45,16 @@ usecase UserRepos safe {
     """
     description string!
   }
+
+example success {
+    input {
+      user = 'superfaceai'
+    }
+
+    result {
```
Comment (smizell) [inline capabilities/vcs/user-repos/profile.supr:54]: I think this needs to be something like:

```
result {
  repos = [
    {
      name = 'one-sdk-js',
      description = 'Superface core client library'
    }
  ]
}
```

File: capabilities/vcs/user-repos/profile.supr [hunk 2]
```diff
@@ -45,4 +45,16 @@ usecase UserRepos safe {
     """
     description string!
   }
+
+example success {
```
Comment (smizell) [inline capabilities/vcs/user-repos/profile.supr:49]: Do you also want to add an example for errors?

**Review by smizell — CHANGES_REQUESTED** [review]
The `vcs/user-repos` profile needs to be tweaked. The others look good.

File: capabilities/vcs/user-repos/profile.supr (hunk 2 above)
Comment (SuperVrata) [reply capabilities/vcs/user-repos/profile.supr:49]: Added it, but not sure the maps really work like that

File: capabilities/vcs/single-file-content/profile.supr [hunk 3]
```diff
@@ -72,4 +72,34 @@ usecase SingleFileContent {
...
+   example success {
+    input {
+      owner = "superfaceai",
+      repo = "one-sdk-js",
+      path = "/README.md",
+      ref = "c5e4d76",
+    }
+
+    result {
+      size = '5',
```
Comment (Jakub-Vacek) [inline capabilities/vcs/single-file-content/profile.supr:85]: size I a number

Comment (SuperVrata) [reply capabilities/vcs/single-file-content/profile.supr]: fixed

**Review by Jakub-Vacek — APPROVED** [review]

**Pull request merged**